skt/templates/report.j2

```diff
@@ -0,0 +1,95 @@
+{# Set some initial variables that we will use throughout the templates. #}
+{% set total_patches = (mergedata.patchwork | length) + (mergedata.localpatch | length) %}
+{% set patch_plural = "patches" if total_patches > 1 else "patch" %}
+Hello,
+
+{% if mergedata['localpatch'] or mergedata['patchwork'] %}
+{# We tested patches against a kernel tree. #}
+We ran automated tests on a patchset that was proposed for merging into this
+kernel tree. The results of these automated tests are provided below.
+{% else %}
+{# We ran baseline tests against a kernel tree. #}
+We ran automated tests on a recent commit from this kernel tree. The results
+of these automated tests are provided below.
+{% endif %}
+
+{# What is the overall job result? #}
+{% if multireport_failed == 1 %}
```

Comment:
I think this would be more readable, if we used the status name, instead of value. E.g.:

    {% if multireport_failed == "MERGE" %}

or:

    {% if multireport_failed == "MULTI_MERGE" %}

or even, if we want a better protection from typos, we could export those numbers as variables and instead write:

    {% if multireport_failed == MULTI_MERGE %}

That would match the Python code! Oh, and I see that we also have greater/lesser comparison, and they would work with this too.